src/test/com/fulcrologic/rad/test_schema_cloud/address.clj

```diff
@@ -0,0 +1,18 @@
+(ns com.fulcrologic.rad.test-schema-cloud.address
```

Comment:
Was there a real need to create new attrs for tests?  aren't there already attrs for tests?